libraries/pbrlib/genglsl/lib/mx_microfacet_specular.glsl

```diff
@@ -301,17 +301,17 @@ vec3 mx_fresnel_airy(float cosTheta, vec3 ior, vec3 extinction, float tf_thickne
 // Parameters for Fresnel calculations.
 struct FresnelData
 {
-    int model;
     vec3 ior;        // In Schlick Fresnel mode these two
     vec3 extinction; // hold F0 and F90 reflectance values
     float exponent;
     float tf_thickness;
     float tf_ior;
+    int model;
```

Comment:
I'm curious why this is last now -- guess this has to do with proper packing?

Reply:
Yes sorting it from high to low - 16, 16, 4, 4, 4, 4,